addons/binding/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/handler/DimmerHandler.java

```diff
@@ -88,6 +93,14 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                 output(ACTION_ZONELEVEL, 100, this.config.getFadeInTime());
             } else if (command.equals(OnOffType.OFF)) {
                 output(ACTION_ZONELEVEL, 0, this.config.getFadeOutTime());
+
+            } else if (command.equals(IncreaseDecreaseType.INCREASE)) {
+                output(ACTION_START_RAISING);
+            } else if (command.equals(IncreaseDecreaseType.DECREASE)) {
+                output(ACTION_START_LOWERING);
+            } else if (command.equals(StopMoveType.STOP)) {
+                output(ACTION_STOP_RAISING_LOWERING);
```

Comment:
Might want to add a `queryOutput(ACTION_ZONELEVEL)` to get an update on where it stopped.